### PR TITLE
code for email alerter, just in case lost

### PR DESCRIPTION
**Pull request by AlexSkowronski2**
Hi guys,

Created a folder and stored the code for the lambda function, just in case it was lost on AWS wanted the backup

Cheers xxxx

### CODE REVIEW

File: heart-rate-alert/lambda.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    SENDER = "trainee.alex.skowronski@sigmalabs.co.uk"
+    RECIPIENT = "three.musketeers.deloton@gmail.com"
+
+    AWS_REGION = "eu-west-2"
+
+    """The subject line for the email."""
+    SUBJECT = "Heart Rate exceeding normal levels"
+
+    """The email body for recipients with non-HTML email clients."""
+    BODY_TEXT = ("You should be aware that your heart rate is exceeding the normal level of {}\r\n"
```
Comment (sebjshaw) [inline heart-rate-alert/lambda.py:15]: How will the normal level for each unique rider be added here? Does this need to be included in the event sent by the EC2?

File: heart-rate-alert/lambda.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                "This email was sent to you by Deloton."
+                )
+
```
Comment (DomLaw82) [inline heart-rate-alert/lambda.py:18]: Clean message👍🏿

File: heart-rate-alert/lambda.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    BODY_TEXT = ("You should be aware that your heart rate is exceeding the normal level of {}\r\n"
+                "This email was sent to you by Deloton."
+                )
+                
+    """The HTML body of the email."""
+    BODY_HTML = f"""<html>
+    <head></head>
+    <body>
+    <h1>Hey {event["user_forename"]} {event["user_surname"]}, </h1>
+
```
Comment (DomLaw82) [inline heart-rate-alert/lambda.py:24]: like the personalisation including the name

**Review by DomLaw82 — APPROVED** [review]

File: heart-rate-alert/lambda.py (hunk 1 above)
Comment (sebjshaw) [reply heart-rate-alert/lambda.py:15]: Later on you've added `{event["limit_heart_rate"]}` should that be here as well?

**Pull request merged**